certbot/CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
-*
+* ACME Renewal Info (ARI) support. https://datatracker.ietf.org/doc/draft-ietf-acme-ari/
+  `certbot renew` will automatically check ARI when using an ACME server that supports it,
+  and may renew early based on the ARI information.
```

Comment:
This may be a good place to add a note about ari overriding renew_before_expiry values set in the config file if it comes first

Reply:
https://github.com/certbot/certbot/issues/10312 tracks making sure this entry is fully fleshed out, so leaving it up to you to merge now or add detail here

Reply:
Thanks for the reminder. Pushed an edit adding detail. Lemme know what you think.